src/components/Progress/index.js

```diff
@@ -90,7 +90,7 @@ const StyledStep = styled(Box, {
   ${({ clickable }) => clickable && clickableStyles}
 `
 
-const Step = () => null
+export const Step = () => null
```

Comment:
If there is no props there is no value to export it no ?

Reply:
I forgot to add proptypes but there is a one prop